app.js

```diff
@@ -0,0 +1,194 @@
+
+// // Easy Going
+ for( let q = 1; q <= 20; q = q + 1) {
+ 	console.log(q);
+ }
+
+// // Get Even
+ for( let q = 0; q <= 200; q++){
+ 	if((q % 2) === 0) { 
+ 		console.log(q)
+
+ 	}
+}
+
+
+// // Excited Kitten
+
+
+for(let i = 1; i < 20; i++) {
+	const meowArr = ["...human...why you taking pictures of me?...", "...the catnip made me do it...","...why does the red dot always get away..."];
+	const randomValue = meowArr[Math.floor(meowArr.length * Math.random())];
+
+	
+ if( i % 2 === 0) { 
+
+ 	console.log(randomValue);
+
+ 	} else { 
+	console.log("Love me, pet me! HSSSSSS!")
+ }
+}
```

Comment:
You've got the right idea here, but your pieces need to be rearranged.  Go back and re-read directions and then double check your code:
• your logic will print "Love me, pet me! HSSSSSS!" for odd numbers only since it's in that `else`.  it's supposed to print it 20 times.
• good job getting the phrase to print on even numbers.  
• I'd reorganize the code a little bit:
  • move the list of quotes outside (before) the for loop.  it's like a "library," you don't need/want to keep redeclaring it
  • since you only need random quote every other time, you should move the code that gets the random quote out of the loop into the if statement so it only runs on even numbers.  as is you're doing unnecessary extra work.